docs/packages.md

```diff
@@ -0,0 +1,34 @@
+---
+id: packages
+title: Packages
+sidebar_label: Packages
+---
+
+## Overview
+
+Dark's alpha package manager allows re-using Dark functions from other users or canvases. Today, only Dark employees can add functions to the package manager. Packages are named by the creator, package, function, and version and are accessible via autocomplete.
+
+In the future, packages will be imported vs. being continually accessible, and community members will be able to contribute directly. If you would like to request or contribute a package today, please reach out via Slack.
+
+## Slack Package
+
+The first package in the Dark package ecosystem is for the Slack API. Use these functions from autocomplete (dark/slack/Slack::), the same way you would any other function.
+
+![Slack](assets/packages/slackpackage.png)
+
+## Upcoming Packages
+
+If you would use one of the following packages, please mention in the community to help us prioritize.
```

Comment:
I'd suggest not listing any, but just asking for what people want.

Reply:
I'd also focus on packaging code that users have already, for them, vs building what we'd need to let them contribute directly just yet.